dxr/static_unhashed/css/filter.css

```diff
@@ -58,12 +58,16 @@
     -webkit-box-shadow: 0px 4px 5px 0px rgba(0,0,0,0.25);
     box-shadow: 0px 4px 5px 0px rgba(0,0,0,0.25);
 }
-.sf-select-options .selector-option-label {
+.selector-option-label {
     display: inline-block;
     vertical-align: top;
+    width: 6rem;
```

Comment:
(I don't know that it's this line causing the problem, but I like to stick with line comments since it keeps threads together.) The addition of the badge column seems to have thrown off some math elsewhere in the stylesheet such that labels don't wrap soon enough anymore:

![screen shot 2016-07-20 at 16 04 45](https://cloud.githubusercontent.com/assets/295816/17002098/d0d9a9a6-4e97-11e6-8a0e-bde7cb686df6.png)

Also, is it possible to make the badge column only as wide as it needs to be to fit the actual showing number of badges?